### PR TITLE
Sample files for LaCrosse Technology View (LTV) devices.

### PR DESCRIPTION
**Pull request by mbruski**
Moved tests/lacrosse/BreezePro_LTV-WSDTH01 directory to tests/lacrosse_ltv
Added tests/lacrosse_ltv/LTV-R1  tests/lacrosse_ltv/LTV_TH3  and  tests/lacrosse/lacrosse_ltv/LTV_WR1

### CODE REVIEW

File: tests/lacrosse_ltv/LTV-R1/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+encoded in the Sensor ID but that remains to be discovered.  Note that the sensor
+ID is fixed (does not change following battery replacement) and should match the
+barcode on the bottom of the sensor.
+
+I have yet to determine the decoding of the flags (lable: unknown) for this sensor.
+Rainfall information is provided as transmitted by the sensor (raw_rain1 & raw_rain2)
+until such time that I understand how to map them to 'rain_mm'.
+
+The sensor used to generate these samples often times repeats a given packet several
+times.  This behaviour is not understood at this time.  I believe the dupes should
```
Comment (Mindavi) [inline tests/lacrosse_ltv/LTV-R1/README.md:30]: FYI: typically dupes are sent for data integrity. If one of the packets has a flipped bit, the others are still usable.

File: tests/lacrosse_ltv/LTV-R1/g002_915M_250k.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{time : @0.159060s, model : LaCrosse-R1, id : 3670818, seq : 7, flags : 0, rain1 : 170, rain2 : 5226, mic : CRC}
```
Comment (Mindavi) [inline tests/lacrosse_ltv/LTV-R1/g002_915M_250k.json:1]: Make sure the output is valid JSON, this is not valid JSON (e.g. generate with `rtl_433 -F json` with the implemented decoder).

**Comment by Mindavi** [comment]
Looks good to me. Please ensure the JSON is updated to be valid JSON and then it's good to go.

**Comment by mbruski** [comment]
I'm not a JSON person, sorry.  What is wrong with the output and what needs to be fixed?Sent from Samsung Galaxy smartphone.
-------- Original message --------From: Rick van Schijndel <notifications@github.com> Date: 11/8/20  11:55  (GMT-05:00) To: merbanan/rtl_433_tests <rtl_433_tests@noreply.github.com> Cc: Mike <aj9x@comcast.net>, Author <author@noreply.github.com> Subject: Re: [merbanan/rtl_433_tests] Sample files for LaCrosse Technology View (LTV) devices. (#369) 
@Mindavi commented on this pull request.



In tests/lacrosse_ltv/LTV-R1/g002_915M_250k.json:
> @@ -0,0 +1 @@
+{time : @0.159060s, model : LaCrosse-R1, id : 3670818, seq : 7, flags : 0, rain1 : 170, rain2 : 5226, mic : CRC}

Make sure the output is valid JSON, this is not valid JSON (e.g. generate with rtl_433 -F json with the implemented decoder).

**Comment by Mindavi** [comment]
JSON is encoded like this: 

```json
{"key":"value"}
```

The quotes are missing in the output here. rtl_433 should generate this correctly with the decoder and the argument `-F json`

**Comment by mbruski** [comment]
I'm not sure what I'm doing differently than you are but this is the output my build generates:

{"time" : "@0.159060s", "model" : "LaCrosse-R1", "id" : 3670818, "seq" : 7, "flags" : 0, "rain1" : 170, "rain2" : 5226, "mic" : "CRC"}
{"time" : "@0.127724s", "model" : "LaCrosse-R1", "id" : 3670818, "seq" : 0, "flags" : 0, "rain1" : 170, "rain2" : 6752, "mic" : "CRC"}
{"time" : "@0.127724s", "model" : "LaCrosse-R1", "id" : 3670818, "seq" : 0, "flags" : 0, "rain1" : 170, "rain2" : 6752, "mic" : "CRC"}
{"time" : "@0.159272s", "model" : "LaCrosse-R1", "id" : 3670818, "seq" : 3, "flags" : 0, "rain1" : 170, "rain2" : 9937, "mic" : "CRC"}

Is this not correct?

Mike

**Comment by Mindavi** [comment]
I will fix it up, no problem. Thanks for the samples!

**Pull request closed without merging**